docs/iris/src/whatsnew/latest.rst

```diff
@@ -258,6 +258,9 @@ This document explains the changes made to Iris for this release
 * `@tkknight`_ updated :ref:`installing_iris` to include a reference to
   Windows Subsystem for Linux. (:pull:`3885`)
 
+* `@tkknight`_ updated homepage to include panels so the links are more
```

Comment:
@tkknight Could I trouble you to make `homepage` a link to that actual homepage, so users can easily go there directly?

Reply:
Change made, see 18391a3